sdks/go/pkg/beam/io/healthcare/fhirio/read.go

```diff
@@ -0,0 +1,87 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fhirio
```

Comment:
Please add a package comment. If the IO is still unfinished/experimental you should also specify that. This package and any exported functionality within it will be visible to users on the next beam release, so keeping it documented is important, even while it's a work-in-progress.

Reply:
Make sure to mark as experimental.

Reply:
👍